doc/ar65.sgml

```diff
@@ -81,13 +82,13 @@ This will delete the module named `sub1.o' from the library, printing an
 error if the library does not contain that module.
 
 
-The `l' command prints a list of all modules in the library. Any module
-names on the command line are ignored.
+The `t' command prints a table of all modules in the library (`l' is deprecated).
```

Comment:
I know what you mean.
Let me tell you about the Wikipedia article of X-Window. A the beginning was "X-Window**s** spelling is wrong". If something is wrong or deprecated, don't put it on the top.